app/src/main/java/com/example/ithaca_transit_android_v2/MainActivity.kt

```diff
@@ -15,15 +16,22 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val end = Coordinate(42.444971674516864, -76.48098092526197)
+        val uid = "E4A0256E-5865-4E9F-8A5A-33747CAC7EBF"
```

Comment:
jw, what is `uid`?

Reply:
I believe this is the user ID

Reply:
yes, we do this for extra logging information to log frequency of searches per user w/o using accounts